MyApplication3/app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -0,0 +1,180 @@
+package com.example.myapplication;
+
+public class MatchData {
+
+    //Auto Data
+    boolean workingAuto = false;
+    boolean moveOutOfZone = false;
+    int ampA = 0;
+    int speakerA = 0;
+    enum contactRobot {
+        None,
+        Allies,
+        Opposition
+    }
+    contactRobot contactBot;
+
+    //Teleop Data
+    boolean broke = false;
+    boolean defense = false;
+    int ampT = 0;
+    int ampSpeaker = 0;
+    int speakerT = 0;
+    int dump = 0;
+    enum noteAquisition {
+        None,
+        Floor,
+        Source,
+        Both
+    }
+    noteAquisition noteAquired;
+
+    //End of Match data
+    enum chain {
+        Nope,
+        Attempted,
+        Failed,
+        Succeded
+    }
+    chain chaining;
+
+    enum trap {
+        Nope,
+        Attempted,
+        Failed,
+        Suceded
+    }
+    trap trapping;
+    enum harmony {
+        Nope,
+        Attempted,
+        Failed,
+        Succeded
+    }
+    harmony harmo;
+    String notes = "";
+    // add human player spotlight data later
+
+
+    //Getters
+
+    //Auto
+
+    public boolean isWorkingAuto() {
+        return workingAuto;
+    }
+
+    public boolean isMoveOutOfZone() {
+        return moveOutOfZone;
+    }
+
+    public int getAmpA() {
+        return ampA;
+    }
+
+    public int getSpeakerA() {
+        return speakerA;
+    }
+
+    public contactRobot getContactBot() {
+        return contactBot;
+    }
+
+    //Teleop
+    public boolean isBroke() {
+        return broke;
+    }
+
+    public boolean isDefense() {
+        return defense;
+    }
+
+    public int getAmpT() {
+        return ampT;
+    }
+
+    public int getAmpSpeaker() {
+        return ampSpeaker;
+    }
+
+    public int getSpeakerT() {
+        return speakerT;
+    }
+
+    public int getDump() {
+        return dump;
+    }
+
+    public noteAquisition getNoteAquired() {   return noteAquired;    }
+
+    //Endgame
+
+    public chain getChaining() {    return chaining;    }
+
+    public trap getTrapping() {     return trapping;    }
+
+    public harmony getHarmo() {     return harmo;     }
+
+    public String getNotes() {
+        return notes;
+    }
+
+    //Setters

```

Comment:
It's usually better to have the getter/setter pairs together. For example, if we need to rename one of them it will be easier to find them all at the same place.